README.md

```diff
@@ -98,8 +98,9 @@ Your brand new file should look something similar to this one:
 ```
 
 Your URL will be `https://linkfree.eddiehub.io/<yourusername>`
+For example: <https://linkfree.eddiehub.io/eddiejaoude> 
 
-For example: <https://linkfree.eddiehub.io/eddiejaoude>
+**Your profile might not be displayed after your PR is merged because we collect some profiles together and deploy the project.**
```

Comment:
Is this not a duplicate with line `109`?

Reply:
My Apologies I skipped that line while reading